pkg/proxy/engines/deltaproxycache.go

```diff
@@ -450,38 +457,45 @@ func DeltaProxyCacheRequest(w http.ResponseWriter, r *http.Request) {
 		rts.Merge(false, ffts)
 	}
 	rts.SetExtents(nil) // so they are not included in the client response json
-	rts.SetStep(0)
-	rdata, err := client.MarshalTimeseries(rts)
+	//rts.SetTimeRangeQuery(&timeseries.TimeRangeQuery{})
```

Comment:
this is ok as-is. the removed lines are not needed after all ts origins are now converted to comment format, and the commented line was needed for some origins during the transition. i've left it here for now just in case it will need to be revived as things evolve in 2.0.